lib/reporting/identity_verification_report.rb

```diff
@@ -282,39 +323,43 @@ def data
           event = row['name']
           user_id = row['user_id']
           success = row['success']
+          gpo_verification_pending = row['gpo_verification_pending'] == '1'
+          in_person_verification_pending = row['in_person_verification_pending'] == '1'
+          fraud_review_pending = row['fraud_review_pending'] == '1'
 
-          event_users[event] << user_id
+          ignore_event_for_user =
+            fraud_review_pending &&
+            EVENTS_TO_IGNORE_IF_FRAUD_REVIEW_PENDING.include?(event)
+
+          users[event] << user_id unless ignore_event_for_user
+          users[sp_key(row['service_provider'])] << user_id if row['service_provider'].present?
```

Comment:
should we maybe break out a different hash? I think it's confusing that sometimes we look up by event, and sometimes we look up by SP

Reply:
We look up by event and "result" already (which is something that is not an event), so I think this is a fair extension of that.